img/a/auth0.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000" style="enable-background:new 0 0 1000 1000" xml:space="preserve"><path d="M753.3 794.5 656.6 500l253.3-182h-313L500.1 23.5h313.2L910.1 318c56.1 170.8-1.7 365.2-156.8 476.5zm-506.6 0 253.3 182 253.4-182.1-253.3-182-253.4 182.1zM90.1 317.9c-59.2 180.2 9.5 371 156.4 476.6v-.1L343.4 500 90.1 317.9l313.1.1L500 23.5H186.9L90.1 317.9z" style="fill:#eb5626"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 309.3 343.8"><path d="M266.31 0H154.65l34.51 107.54h111.66l-90.34 64.13L245 279.82c58.15-42.36 77.15-106.47 55.83-172.28zM8.48 107.54h111.66L154.65 0H43L8.48 107.54c-21.33 65.81-2.32 129.92 55.83 172.28l34.51-108.15zm55.83 172.28 90.34 64 90.34-64-90.34-65.06z"/></svg>
```
The [Auth0 Press Kit](https://auth0.com/press) uses the black version of the logo now